README.md

```diff
@@ -33,6 +33,9 @@ The VPN server is deployed in a selected multizone (MZR) region and VPC.
 - **VPN Server authentication**: The VPN server certificate is needed at the time of provisioning of the VPN server. The server certificate CRN in the IBM Cloud Secrets Manager is required.
 - **VPN Client authentication**: The client authentication method is required at the time of provisioning of the VPN server. The VPN server supports "username" based authentication. There is no support for the "certificate" based authentication at this point of time.
 - **Split tunnel mode**: This module provisions a client-to-site VPN with the split tunnel mode. When the VPN connection is set up, an encrypted tunnel is created over the internet to the VPN server. The split tunnel mode supports sending private traffic that is destined to the VPC inside the VPN tunnel and sending public traffic (internet traffic) outside the VPN tunnel.
+- **Accessing the VPC clusters**:
+  - Through the private service endpoint: Set up `vpn_server_routes` as mentioned [here](https://cloud.ibm.com/docs/openshift?topic=openshift-access_cluster#access_private_se).
+  - Through the Virtual Private Endpoint (VPE) gateway: Set up `vpn_server_routes` as mentioned [here](https://cloud.ibm.com/docs/openshift?topic=openshift-access_cluster#vpc_vpe).
```

Comment:
```suggestion
- **Access to the VPC clusters**: You can allow authorized cluster users to access your VPC cluster through the private service endpoint or the Virtual Private Endpoint Gateway. Specify the access in the `vpn_server_routes` input. For more information, see [Accessing clusters through the private cloud service endpoint](https://cloud.ibm.com/docs/openshift?topic=openshift-access_cluster#access_private_se) and [through the Virtual Private Endpoint Gateway](https://cloud.ibm.com/docs/openshift?topic=openshift-access_cluster#vpc_vpe).
```

suggestion for a different way of organizing and presenting the information.

Reply:
That doesn't look right, its not about the users. So basically what we are doing here is to add CIDR ranges to the `vpn_server_route` so that the VPN can access the cluster either by vpe or private server endpoint.

Reply:
how about

> **Access to the VPC clusters**: Define CIDR blocks to allow VPN access to your VPC cluster through either the private service endpoint or the Virtual Private Endpoint Gateway. Specify the block in the `vpn_server_routes` input. For more information about the endpoints, see [Accessing clusters through the private cloud service endpoint](https://cloud.ibm.com/docs/openshift?topic=openshift-access_cluster#access_private_se) and [through the Virtual Private Endpoint Gateway](https://cloud.ibm.com/docs/openshift?topic=openshift-access_cluster#vpc_vpe).

Reply:
Looks good, just a minor change instead of "For more information about the endpoints" I changed that to "For more information about the routes" as they are the values for the VPN server routes. I have updated the PR.